clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -165,4 +165,17 @@
       </xs:annotation>
     </xs:attribute>
   </xs:attributeGroup>
+
+  <xs:element name="clustered-store" type="tc:clustered-store-type" substitutionGroup="eh:service-configuration"/>
```

Comment:
This is to be consistent with other store level configurations - have a look at the default Ehcache XSD.